examples/vercel-edge-runtime/src/client.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  createTRPCClient,
+  createTRPCClientProxy,
+  httpBatchLink,
+  loggerLink,
+} from '@trpc/client';
+import fetch from 'node-fetch';
+import type { AppRouter } from './router';
+
+// polyfill
+globalThis.fetch = fetch as any;
+
+const sleep = (ms = 100) => new Promise((resolve) => setTimeout(resolve, ms));
+
+async function main() {
+  const url = 'http://127.0.0.1:8787';
+
+  const client = createTRPCClient<AppRouter>({
+    links: [loggerLink(), httpBatchLink({ url })],
+  });
+
+  const proxy = createTRPCClientProxy(client);
+
```

Comment:
We have a newer way of doing this with `createTRPCProxyClient`

https://trpc.io/docs/vanilla

Reply:
Copied from the cf-workers example, will update there too